app/_layouts/extension.html

```diff
@@ -257,6 +421,32 @@
 {{ plugin_global_config_admin_api }}
 
 {% endnavtab %}
+{% unless page.params.k8s_compatible == false %}
+{% navtab Kubernetes %}
+
+Create a [KongPlugin](https://github.com/Kong/kubernetes-ingress-controller/blob/main/docs/guides/using-kongplugin-resource.md)
+resource and label it as global:
+
+```yaml
+apiVersion: configuration.konghq.com/v1
+kind: KongPlugin
+metadata:
+  name: <global-{{page.params.name}}>
+  labels:
+    global: \"true\"
+{% if config_required_fields_k8s == 'config: ' %}config:
+  <optional_parameter>: <value>{% else %}{{ config_required_fields_k8s }}{% endif %}
+plugin: {{page.params.name}}
+```
+
+<div class="alert alert-ee blue">
+  <strong>Note:</strong> If you want the plugin to be available cluster-wide,
+  create the resource as a <code>KongClusterPlugin</code> instead of
+  <code>KongPlugin</code>.
+</div>
```

Comment:
As of 0.10 (not yet released) we'll start requiring KongClusterPlugins for global plugins:

```suggestion
Create a [KongClusterPlugin](https://github.com/Kong/kubernetes-ingress-controller/blob/main/docs/references/custom-resources.md#kongclusterplugin)
resource and label it as global:

```yaml
apiVersion: configuration.konghq.com/v1
kind: KongClusterPlugin
metadata:
  name: <global-{{page.params.name}}>
  labels:
    global: \"true\"
{% if config_required_fields_k8s == 'config: ' %}config:
  <optional_parameter>: <value>{% else %}{{ config_required_fields_k8s }}{% endif %}
plugin: {{page.params.name}}
```
```

Reply:
Current target is the 16th. However, KongClusterPlugins are already available, so no harm in using them in advance--they aren't available in versions older than 0.8, but that's been out for a while now (since March).